### ISSUE TITLE
Torrent paused after download

### ISSUE THREAD

**Issue by Berbe** [transmission/transmission#7160]
### What is the issue?

After a torrent finished downloading, it systematically stops, even though I set :
```YAML
  "ratio-limit": 10,
  "ratio-limit-enabled": true,
```

Transmission version derives from `9ff95d162e51d0dbacff242e7f6c7483847d61aa` with a couple fixes added on top: see [`Berbe/transmission#493d60731ba7dd1ad75a069cf8e5568e643825a6`](https://github.com/Berbe/transmission/commits/493d60731ba7dd1ad75a069cf8e5568e643825a6/)

### Which application of Transmission?

transmission-daemon

### Which version of Transmission?

9ff95d162e51d0dbacff242e7f6c7483847d61aa

**Comment by Berbe** [comment]
Yes, I am talking about torrent getting paused as soon as _download_ is finished not _upload/seeding_, hence this bug report.

**Comment by Berbe** [comment]
Any news on this?

**Comment by Berbe** [comment]
I narrowed it down to the fact this happens when temporary & downloads directories are individual Docker mounts.
This must be related to how directories are handled/watched and how container volume mounts interfere with that.

**Comment by Berbe** [comment]
Another piece of information: the bug does not trigger if `incomplete-dir-enabled` is set to `false`, making completed downloads not moving from one directory to another: another indication this is related to directories handling.

# Reproduction
## Transmission configuration
* `download-dir`: `/mnt/download`
* `incomplete-dir`: `/mnt/incomplete`
* `incomplete-dir-enabled`: `true`
* Adapt `rpc-whitelist` or `rpc-whitelist-enabled` to container environment
I personally set `rpc-whitelist-enabled` to `false`
* `port-forwarding-enabled`: `false`; useless & spammy in container

## Host directories
```
data/
├── download
└── incomplete
```

## Container configuration
* Mount separate volumes (host bind-mounts) in container:
Pay attention to provide proper user & group ownership or permissions so that Transmission can read & write there
  * `data/download` -> `/mnt/download`
  * `data/incomplete` -> `/mnt/incomplete`

## Problem circumvention
If the following mount is used, the problem does not trigger:
* `data` -> `/mnt`

Any combination of that mount with the `download` or the `incomplete` one yields the problem again

**Comment by abubaca4** [comment]
Look like solved with https://github.com/transmission/transmission/pull/8175 in 4.1.0
